tests/sentry/event_manager/test_event_manager.py

```diff
@@ -1540,6 +1541,133 @@ def test_transaction_event_span_grouping(self) -> None:
         # the basic strategy is to simply use the description
         assert spans == [{"hash": hash_values([span["description"]])} for span in data["spans"]]
 
+    @override_options({"transactions.do_post_process_in_save": True})
```

Comment:
What value does this test add in contrast with the next one? I think they're the same but the next one tests for the two calls.

Reply:
there is a comment below, but the next test ensures that the functions get called with the correct mocks, while this one ensures that the functions themselves don't error / run correctly. (didn't see a great way to validate that the side effects those functions call), so have two.